src/components/form/input.js

```diff
@@ -82,7 +82,7 @@ const StyledInput = styled.input`
     font-size: var(--text-size-s);
     padding: 1rem 1rem 1rem 0.8rem;
     height: ${(props) => props.height || '40px'};
-    width: 100%;
+    width: ${(props) => (props.password_icon ? '90%' : '100%')};
```

Comment:
```suggestion
    width: ${(props) => (props.password_icon ? '90%' : '100%')};
```
Can we do something like `calc(100% - icon_width)` ?

Reply:
Yes, refactored here.